src/components/shared/MultiSelect/index.tsx

```diff
@@ -223,6 +236,28 @@ const MultiSelect = ({
     );
   }, [initialValues, originalOptions]);
 
+  const sortSelectedTags = (order: 'asc' | 'desc' | string) => {
+    switch (order) {
+      case 'desc':
+        return [
+          ...selected
+        ].sort((a: { value: string }, b: { value: string }) =>
+          b.value.localeCompare(a.value)
+        );
+      case 'asc':
+        return [
+          ...selected
+        ].sort((a: { value: string }, b: { value: string }) =>
+          a.value.localeCompare(b.value)
+        );
+      default:
+        return [
+          ...selected.filter(tag => tag.value === order),
+          ...selected.filter(tag => tag.value !== order)
+        ];
+    }
+  };
+
```

Comment:
I think we don't want to default to ASC for sorting.  In almost all existing cases, we want to leave the tag in the order they are selected, so no sort necessary.